clang/test/CodeGen/WebAssembly/wasm-externref-novec.c

```diff
@@ -0,0 +1,18 @@
+// RUN: %clang_cc1 -O2 -triple wasm32-unknown-unknown-wasm -target-feature +reference-types -emit-llvm -o - %s | FileCheck %s
+
+// From issue 69894. Reftypes need to be marked as not valid as vector elements.
```

Comment:
If you would like to refer to an issue, I think including the URL is more convenient: `https://github.com/llvm/llvm-project/issues/69894`